File: preview/marketplace/available_add_on_extension/fetch-default/fetch-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Preview.Marketplace.AvailableAddOn;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var availableAddOnExtension = AvailableAddOnExtensionResource.Fetch(
            pathAvailableAddOnSid: "XBXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
            pathSid: "XFXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
        );

        Console.WriteLine(availableAddOnExtension.FriendlyName);
    }
}

File: preview/marketplace/available_add_on_extension/fetch-default/fetch-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

available_add_on_extension = @client.preview
  .marketplace
  .available_add_ons('XBXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
  .extensions('XFXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
  .fetch

puts available_add_on_extension.friendly_name
